LICENSE

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 dc-square GmbH
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Don't we need to fill in the template values?

Reply:
No, this is just a template which results in the HEADER file. The appendix describes the usage of the license but is not supposed to be changed.